components/brave_wallet/common/eth_sign_typed_data_helper.cc

```diff
@@ -194,12 +185,12 @@ EthSignTypedDataHelper::EncodeData(const std::string& primary_type_name,
 
 // Encode each field of a custom type, if a field is also a custom type it
 // will call EncodeData recursively until it reaches an atomic type
-std::optional<std::vector<uint8_t>> EthSignTypedDataHelper::EncodeField(
-    const std::string& type,
-    const base::Value& value) const {
+std::optional<EthSignTypedDataHelper::Eip712HashArray>
+EthSignTypedDataHelper::EncodeField(const std::string& type_string,
```

Comment:
Not sure if possible, but if possible, change to `std::string_view`

Reply:
Made this file to use std::string_view